packages/vite-config/src/vitest-config.ts

```diff
@@ -7,6 +7,7 @@ export default mergeConfig(config, {
     include: ['@vitest/utils', 'vitest/browser'],
   },
   test: {
+    watch: false,
```

Comment:
Default behavior for Vitest is to watch, which would makes it appear like it hangs forever if you just run `pnpm test` in the root directory. For all of our Vitest projects, make the default behavior not to watch. You can still run `vitest watch` or `vitest dev` if you want to use it locally.